app/models/redirect.rb

```diff
@@ -12,6 +12,6 @@ def url_is_valid
   end
 
   def visit!
-    update_attributes!(last_visited_at: Time.now, :visits => visits + 1)
+    update!(last_visited_at: Time.now, :visits => visits + 1)
```

Comment:
I'm not positive what we do with "visits", but I'm wondering if it might be beneficial to move this over to a real analytics platform? Maybe something self-hosted and not GA? @mileszs and @armilam what do you think?

Reply:
I have no idea what visits is here. I'm not sure what we want to do with visit data, but if there is a purpose, then I agree a real platform might be in order if we can get it free or super cheap.